src/app/query/bitcoin/bitcoin-client.ts

```diff
@@ -30,12 +30,29 @@ class AddressApi {
   }
 }
 
+class FeeEstimatesApi {
+  configuration: Configuration;
+
+  constructor(configuration: Configuration) {
+    this.configuration = configuration;
+  }
```

Comment:
```ts
  constructor(public configuration: Configuration) {}
```
This is the same as this btw

Reply:
My build broke here so reverted back...

![Screen Shot 2022-11-22 at 12 38 27 PM](https://user-images.githubusercontent.com/6493321/203396183-be3ac9be-b1c3-448a-8eb2-20fc1b29073c.png)